src/composables/auth.ts

```diff
@@ -122,10 +122,11 @@ export const useAuth = createCustomScopedComposable(() => {
   /**
    * Setting/Resetting the password key logs the user in/out.
    */
-  function setEncryptionKey(newEncryptionKey: CryptoKey | null) {
+  async function setEncryptionKey(newEncryptionKey: CryptoKey | null) {
     encryptionKey.value = newEncryptionKey;
     if (IS_EXTENSION) {
       if (newEncryptionKey) {
+        await watchUntilTruthy(secureLoginTimeout);
```

Comment:
Make sure that the `secureLoginTimeout` has a value, since it is an encrypted value we need to make sure that it is decrypted and set.